docs/application/dotnet/guides/nui/text.md

```diff
@@ -0,0 +1,590 @@
+label.FontStyle = fontStyle;
+label.PointSize = 12.0f;
+```
+
+If no font is specified, default styles are used, and a suitable font for displaying the text label is automatically selected from the platform. However, the automatically-selected font may not render all the characters contained within the text label. For example, Latin fonts often do not provide Arabic glyphs.
+
+### Font Styles
+
+Setting a font size programmatically is not ideal for applications that support multiple screen resolutions, and for platforms that support multiple logical font sizes. In addition, making systemwide changes to your font settings override the font sizes that have been programmatically set.
+
+A more flexible approach is to prepare various JSON stylesheets and request a different style for each platform. The [Tizen.NUI.NUIApplication](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.NUIApplication.html) class has constructors that takes a stylesheet argument:
+
+```csharp
+class Example : NUIApplication
+
+Example example = new Example("example-path/example.json");
+```
+
+To change the font style for standard text controls, use the following JSON syntax:
+
+```csharp
+{
+   "styles":
+   {
+      "textlabel":
+      {
+         "fontFamily": "FreeSerif",
+         "fontStyle":
+         {
+            "weight": "bold",
+            "slant": "italic"
+         },
+         "pointSize": 8
+      }
+   }
+}
+```
+
+However, the same `pointSize` is unlikely to be suitable for all text controls in an application. To define custom styles for existing controls, set a style name for each case, and provide a style override in a JSON stylesheet.
+
+You can provide further flexibility for the various screens by mapping the logical size to a physical size in the stylesheet.
+
+<a name="align"></a>
+### Align Text
+
+To align the text in a text label:
+
+-   To enable text wrapping, use the `MultiLine` property of the [Tizen.NUI.BaseComponents.TextLabel](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.BaseComponents.TextLabel.html) class:
+
+    ```csharp
+    label.MultiLine = true;
+    ```
+
+-   To align the text horizontally to the beginning, center, or end of the available area, set the `HorizontalAlignment` property of the `Tizen.NUI.BaseComponents.TextLabel` class with the corresponding value of the [Tizen.NUI.HorizontalAlignment](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.HorizontalAlignment.html) enumeration:
+
+    ```csharp
+    label.HorizontalAlignment = HorizontalAlignmentType.Begin;
+    label.HorizontalAlignment = HorizontalAlignmentType.Center;
+    label.HorizontalAlignment = HorizontalAlignmentType.End;
+    ```
+
+    The following table summarizes the avaiable values of the `Tizen.NUI.HorizontalAlignment` enumeration for both left-to-right (Latin) and right-to-left (Arabic) script. In addition, for the illustrated examples, it is  assumed that the label size is greater than the minimum required size:

```

Comment:
Spell mistake: 
Kindly chnage 
avaiable
to 
available
The following table summarizes the available values of the `Tizen.NUI.HorizontalAlignment` enumeration for both left-to-right (Latin) and right-to-left (Arabic) script.